lib/slack_coder/users/supervisor.ex

```diff
@@ -28,6 +28,10 @@ defmodule SlackCoder.Users.Supervisor do
     end
   end
 
+  def stop_user(user) do
+    Supervisor.terminate_child(__MODULE__, "User-#{String.downcase(user.slack)}-#{String.downcase(user.github)}")
```

Comment:
You also have to call `delete_child`... I think I've done this somewhere but have to find it again... Otherwise the supervisor still tracks the dead child 👻